appveyor.yml

```diff
@@ -15,7 +15,7 @@ install:
   - set PATH=%CONDA%;%CONDA%\Scripts;%PATH%
   - conda config --set always_yes yes
   - conda update conda
-  - conda install python=%PYTHON% numpy scipy=1.0.1 mkl==2018.0.2 Cython sphinx pip --quiet
+  - conda install python=%PYTHON% numpy pandas scipy=1.0.1 mkl==2018.0.2 Cython sphinx pip --quiet
```

Comment:
I think it is overkill to introduce this dependency as I understand that it is just used to load csv file in data.py. It should be replaced by numpy equivalent.

Reply:
Agreed. Updated files accordingly.